app/src/androidTest/java/org/mozilla/focus/activity/WebControlsTest.kt

```diff
@@ -38,12 +44,14 @@ class WebControlsTest {
         webServer.start()
         featureSettingsHelper.setCfrForTrackingProtectionEnabled(false)
         featureSettingsHelper.setNumberOfTabsOpened(4)
+        Intents.init()
```

Comment:
Is this needed? I know there's similar test on Fenix but I don't remember needing to initialize anything.

Reply:
Without it the intent can't be verified.
`java.lang.NullPointerException: Intents not initialized. Did you forget to call init()?`